src/client/utils/error-monitor/error-monitor.ts

```diff
@@ -17,6 +19,12 @@ export function addErrorMonitor() {
       console.log(JSON.stringify(err));
     }
 
+    Qajax({
+      method: "POST",
+      url: 'error',
+      data: err
+    }).done();
```

Comment:
in defiance of what is normally considered good we probably should not call `.done()` here. If the post fails, let's just swallow the error.